### PR TITLE
fix(security) Secure upload images

### PR DESCRIPTION
**Pull request by loiclau**

### CODE REVIEW

File: www/class/centreonFileManager.php [hunk 1]
```diff
@@ -0,0 +1,175 @@
+<?php
+/**
```
Comment (kduret) [inline www/class/centreonFileManager.php]: Missing Centreon License header

File: www/class/centreonImageManager.php [hunk 2]
```diff
@@ -0,0 +1,187 @@
+<?php
+/**
```
Comment (kduret) [inline www/class/centreonImageManager.php]: Missing Centreon License header

File: www/class/iFileManager.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?php
+/**
```
Comment (kduret) [inline www/class/iFileManager.php]: Missing Centreon License header

File: www/class/centreonImageManager.php [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+        parent::upload();
+        if ($insert) {
+            $img_ids[] = $this->insertImg(
+                $this->destinationPath,
+                $this->destinationDir,
+                $this->newFile,
+                $this->comment
+            );
+            return $img_ids;
+        }
```
Comment (kduret) [inline www/class/centreonImageManager.php]: maybe return false ?

File: www/class/centreonImageManager.php [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+        } elseif ($img_info['dir_alias'] != $this->destinationDir) {
+            $old = $this->mediaPath . $img_info['dir_alias'] . '/' . $img_info["img_path"];
+            $new = $this->mediaPath . $this->destinationDir . '/' . $img_info["img_path"];
+            $this->moveImage($old, $new);
+        }
+
+        //update relation
+        $query = "UPDATE view_img_dir_relation SET dir_dir_parent_id  = '" . $dirId .
+            "' WHERE img_img_id = '" . $imgId . "'";
+        $pearDB->query($query);
```
Comment (kduret) [inline www/class/centreonImageManager.php]: Maybe return true ?

File: www/include/options/media/images/formImg.php [hunk 6]
```diff
@@ -216,7 +220,7 @@
 }
 $action = $form->getSubmitValue("action");
 
-if ($valid) {
+if (is_array($valid)) {
```
Comment (kduret) [inline www/include/options/media/images/formImg.php]: strange validation. I think methods need to return "false" if update or upload doesn't work

**Comment by adr-mo** [comment]
I've tried to upload php and js using image format (extensions and headers). Even if file is uploaded, I could not figure out how to execute it.

Everything looks good to me.

**Pull request merged**